doc/dev/testing/travis-build.md

```diff
@@ -32,7 +32,8 @@ The Go, Ansible, and Helm tests then differ in what tests they run.
     2. Check that all source files have a license.
     3. Check that all error messages start with a lower case alphabetical character and do not end with punctuation, and log messages start with an upper case alphabetical character.
     4. Make sure the repo is in a clean state (this is particularly useful for making sure the `Gopkg.lock` file up to date after `dep ensure`).
-2. Run [unit tests][unit].
+2. Run unit tests.
+    1. Run `make test` or `go test -count=1 -short ./commands/... ./pkg/...`.
```

Comment:
Nit: Think the go test command is not needed, as user can see that in the Makefile target.

Reply:
Removed the go command.